src/com/ibm/ServerWizard2/ServerWizard2.java

```diff
@@ -149,6 +150,9 @@ private static void getPreferencesForUpdateMode() {
 		try {
 			Properties p = new Properties();
 			File f = new File(ServerWizard2.PROPERTIES_FILE);
+			Display display = new Display();
```

Comment:
As discussed, these directory dialogs are not needed anymore. What purpose were they achieving?